scripts/deploy/contracts/InvestorVesting.ts

```diff
@@ -45,11 +55,11 @@ export const deploy = async (deployer, setAddresses) => {
   setAddresses({ InvestorVesting: investorVesting.address });
   console.log("Verifying InvestorVesting on Etherscan");
   // Wait for a bit for etherscan to pick up the contract.
-  await sleep(5000);
+  await sleep(15000);
```

Comment:
Instead of doing this, line 52 can be changed to wait for more notifications. 6 is usually a good number.

Reply:
Neat!